src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
```

Comment:
The function name `makeCalculator` is appropriate as it describes the action of creating a calculator object. Good naming choice here.